doc/api/util.md

```diff
@@ -68,8 +68,7 @@ exports.puts = util.deprecate(() => {
 It returns a modified function which warns once by default.
 
 If `--no-deprecation` is set then this function is a NO-OP.  Configurable
-at run-time through the `process.noDeprecation` boolean (only effective
-when set before a module is loaded.)
+at run-time through the `process.noDeprecation` boolean.
```

Comment:
Yes, that looks good (https://github.com/nodejs/node/pull/6683#discussion_r62877048)